### PR TITLE
Update for JDSOL 2.0-RC2 part one (there is no part two)

### PR DESCRIPTION
**Pull request by markt-asf**
There is no API JAR (nor a TCK) for this spec so there will be no part 2

Signed-off-by: Mark Thomas <markt@apache.org>

## Specification PR template
When creating a specification project release review, create two PRs with the content divided as follows.

Include the following in PR#1:
- [X] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [X] Specification PDF in the form of wombat-spec-x.y.pdf
- [X] Specification HTML in the form of wombat-spec-x.y.html
- [X] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [X] For a Progress Review, that sufficient progress has been made on
      a Compatible Implementation and TCK, to ensure that the spec is
      implementable and testable.
- [ ] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [ ] The URL of the OSSRH staging repository for the api, javadoc:
      <add URL here>
- [ ] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      <add URL here>
- [ ] The URL of the compatibility certification request issue:
      <add URL here>

### CODE REVIEW

File: debugging/2.0/_index.md [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
-* [Jakarta Debugging Support for Other Languages 2.0 Release Record]()
+* [Jakarta Debugging Support for Other Languages 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jsp/releases/2.0-debugging-support-other-languages)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Debugging Support for Other Languages 2.0 Specification Document]() (PDF)
-* [Jakarta Debugging Support for Other Languages 2.0 Specification Document]() (HTML)
-* [Jakarta Debugging Support for Other Languages 2.0 Javadoc](./apidocs)
-* [Jakarta Debugging Support for Other Languages 2.0 TCK]()([sig](),[sha](),[pub]())
-* Maven coordinates
-  * [jakarta.debugging:jakarta.debugging-api:jar:2.0.0]()
+* [Jakarta Debugging Support for Other Languages 2.0 Specification Document](./jdsol-sepc-2.0-RC2.pdf) (PDF)
```
Comment (kwsutter) [inline debugging/2.0/_index.md:13]: typo...  should be -spec-, not -sepc-...

File: debugging/2.0/_index.md [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
+* [Jakarta Debugging Support for Other Languages 2.0 Specification Document](./jdsol-sepc-2.0-RC2.html) (HTML)
```
Comment (kwsutter) [inline debugging/2.0/_index.md:14]: typo...  should be -spec-, not -sepc-...

File: debugging/2.0/_index.md (hunk 1 above)
Comment (markt-asf) [reply debugging/2.0/_index.md:13]: Fixed.

File: debugging/2.0/_index.md (hunk 2 above)
Comment (markt-asf) [reply debugging/2.0/_index.md:14]: Fixed.

**Comment by markt-asf** [comment]
Typos fixed. Confirmed image files are embedded in HTML so I've removed them. Forced updating on the way...

**Pull request merged**